core/src/display_object/edit_text.rs

```diff
@@ -1583,11 +1583,6 @@ impl<'gc> TDisplayObject<'gc> for EditText<'gc> {
             .as_node()
             .duplicate(context.gc_context, true)
             .document();
-
-        let mut new_layout = Vec::new();
```

Comment:
btw did you find out anything about this? Maybe there should be a note left about this somewhere...

Reply:
This code was introduced in d7a257f93faf8a585d1d00af02e1682014d9705e, and used to be:
```rs
text.layout = text.layout.map(|l| l.read().duplicate(context.gc_context));
```
which seems equivalent except the assignment to `text.layout`, that makes it effective.
@kmeisthax Maybe do you have an insight about this code change. Correct me if I'm wrong, but it feels to me that the previous code is still correct and should be re-introduced.